pkg/operation/care/health.go

```diff
@@ -301,6 +301,8 @@ func init() {
 	utilruntime.Must(err)
 	versionConstraintGreaterEqual132, err = semver.NewConstraint(">= 1.32")
 	utilruntime.Must(err)
+	versionConstraintGreaterEqual132, err = semver.NewConstraint(">= 1.36")
```

Comment:
```suggestion
	versionConstraintGreaterEqual136, err = semver.NewConstraint(">= 1.36")
```

However, see earlier comment: I guess you have to change to 1.37 here